src/core_modules/capture-core/components/WidgetEnrollment/MiniMap/MiniMap.component.tsx

```diff
@@ -27,12 +28,12 @@ const MiniMapPlain = ({
     refetchTEI,
     onError,
     classes,
-}: MiniMapProps) => {
+}: Props) => {
     const [isOpenMap, setOpenMap] = useState(false);
-    const { updateMutation } = useUpdateEnrollment(refetchEnrollment, refetchTEI, onError);
+    const { updateMutation = () => { /* empty by design */ } } = useUpdateEnrollment(refetchEnrollment, refetchTEI, onError) ?? {};
```

Comment:
Please remove the comment you added. The code should look like this: 
`const { updateMutation } = useUpdateEnrollment(refetchEnrollment, refetchTEI, onError);`